### PR TITLE
fix upload button in IE11

### PR DESCRIPTION
**Pull request by IgorZiegler**
It is very very strange and very hard to search this bug. This magic is really work.

### CODE REVIEW

**Comment by IgorZiegler** [comment]
#137

File: js/lib/resumable.js/resumable.js [hunk 1]
```diff
@@ -857,6 +857,11 @@
 					input.setAttribute('type', 'file');
 					input.style.display = 'none';
 					domNode.addEventListener('click', function(){
+						for (key in input) {
```
Comment (knutole) [inline js/lib/resumable.js/resumable.js]: use `var key`, so as not to create a global.

File: js/lib/resumable/resumable.js [hunk 2]
```diff
@@ -857,6 +857,11 @@
 					input.setAttribute('type', 'file');
 					input.style.display = 'none';
 					domNode.addEventListener('click', function(){
+						for (key in input) {
```
Comment (knutole) [inline js/lib/resumable/resumable.js]: var key

Comment (IgorZiegler) [inline js/lib/resumable/resumable.js]: Ohh great thanks!

File: js/lib/resumable.js/resumable.js [hunk 3]
```diff
@@ -857,6 +857,11 @@
 					input.setAttribute('type', 'file');
 					input.style.display = 'none';
 					domNode.addEventListener('click', function(){
+						for (var key in input) {
```
Comment (knutole) [inline js/lib/resumable.js/resumable.js:860]: could be race condition. try this:

```
setTimeout(function () {
    input.style.opacity = 0;
    input.style.display='block';
    input.focus();
}, 100);
```

**Pull request merged**